cyclops-ui/src/components/k8s-resources/common/PodTable/PodTable.tsx

```diff
@@ -160,49 +248,78 @@ const PodTable = ({ pods, namespace }: Props) => {
       <div style={{ width: "400px" }}>
         <h3>{pod.name} actions</h3>
         <Divider style={{ margin: "8px" }} />
-        <Button
-          style={{ width: "60%", margin: "4px" }}
-          onClick={function () {
-            axios
-              .get(
-                "/api/resources/pods/" +
-                  namespace +
-                  "/" +
-                  pod.name +
-                  "/" +
-                  pod.containers[0].name +
-                  "/logs",
-              )
-              .then((res) => {
-                if (res.data) {
-                  let log = "";
-                  res.data.forEach((s: string) => {
-                    log += s;
-                    log += "\n";
-                  });
-                  setLogs(log);
-                } else {
-                  setLogs("No logs available");
-                }
-              })
-              .catch((error) => {
-                setError(mapResponseError(error));
-              });
-
-            setLogsModal({
-              on: true,
-              namespace: namespace,
-              pod: pod.name,
-              containers: pod.containers,
-              initContainers: pod.initContainers,
-            });
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "row",
+            alignItems: "center",
+            justifyContent: "center",
           }}
         >
-          <h4>
-            <ReadOutlined style={{ paddingRight: "5px" }} />
-            View Logs
-          </h4>
-        </Button>
+          <Button
+            style={{ width: "60%", margin: "4px" }}
+            onClick={function () {
+              axios
+                .get(
+                  "/api/resources/pods/" +
+                    namespace +
+                    "/" +
+                    pod.name +
+                    "/" +
+                    pod.containers[0].name +
+                    "/logs",
+                )
+                .then((res) => {
+                  if (res.data) {
+                    let log = "";
+                    res.data.forEach((s: string) => {
+                      log += s;
+                      log += "\n";
+                    });
+                    setLogs(log);
+                  } else {
+                    setLogs("No logs available");
+                  }
+                })
+                .catch((error) => {
+                  setError(mapResponseError(error));
+                });
+
+              setLogsModal({
+                on: true,
+                namespace: namespace,
+                pod: pod.name,
+                containers: pod.containers,
+                initContainers: pod.initContainers,
+              });
+            }}
+          >
+            <h4>
+              <ReadOutlined style={{ paddingRight: "5px" }} />
+              View Logs
+            </h4>
+          </Button>
+          <Button
```

Comment:
Can we make the buttons go one under the other? There will be a third button for pod manifest, and it will probably look better if they are one under the other as opposed to being in the same line.

Reply:
resolved in e6dfca2